boot-qemu.sh

```diff
@@ -1,12 +1,19 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Just curious, why?

Reply:
eh, probably not needed, will drop.
sh on my host is symlinked to bash anyways, though it's not my default shell...hmm...

Reply:
On debian systems sh is symlinked to dash, so any bashisms won't work.